component/async/kafka/group/group.go

```diff
@@ -56,7 +56,7 @@ func (f *Factory) Create() (async.Consumer, error) {
 
 	cc := kafka.ConsumerConfig{
 		Brokers:      f.brokers,
-		Buffer:       0,
+		Buffer:       config.ChannelBufferSize,
```

Comment:
Might be safer to do `config.ChannelBufferSize + 1` but I wasn't sure so left as-is, if you feel otherwise let me know and I'll change

Reply:
It seems to make little difference in the scenario where the process function is lower that the number of messages added to the channel.

Reply:
Can you please explain? I don't understand what you mean... 😳